lighthouse-core/gather/drivers/driver.js

```diff
@@ -492,7 +492,8 @@ class Driver {
   beginEmulation() {
     return Promise.all([
       emulation.enableNexus5X(this),
-      emulation.enableNetworkThrottling(this)
+      emulation.enableNetworkThrottling(this),
+      //emulation.enableCPUThrottling(this),
```

Comment:
Worth this PR including a one-liner comment to the README that mentions that requirement for real phones?